flight/odlc_boundaries/search_path.py

```diff
@@ -0,0 +1,204 @@
+"""
+Functions for generating search paths to cover an area for finding the standard odlc objects
+"""
+
+import logging
+import asyncio
+import utm
+from shapely.geometry import Polygon
+from execute import move_to
```

Comment:
Local imports should be relative to root directory
```suggestion
from flight.odlc_boundaries.execute import move_to
```